pkgs/applications/virtualization/looking-glass-client/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, SDL2, SDL, SDL2_ttf, openssl, spice_protocol, fontconfig
+, libX11, freefont_ttf
+}:
+
+stdenv.mkDerivation rec {
+  name = "looking-glass-client-${version}";
+  version = "a10";
+
+  src = fetchFromGitHub {
+    owner = "gnif";
+    repo = "LookingGlass";
+    rev = "a10";
+    sha256 = "10jxnkrvskjzkg86iz3hnb5v91ykzx6pvcnpy1v4436g5f2d62wn";
+  };
+
+  buildInputs = [ pkgconfig SDL SDL2 SDL2_ttf openssl spice_protocol fontconfig
+    libX11 freefont_ttf ];
+
+  enableParallelBuilding = true;
+
+  sourceRoot = "source/client";
+
+  installPhase = ''
+    mkdir -p $out
+    mv bin $out/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A KVM Frame Relay (KVMFR) implementation";
+    longDescription = ''
+      Looking Glass is an open source application that allows the use of a KVM
+      (Kernel-based Virtual Machine) configured for VGA PCI Pass-through
+      without an attached physical monitor, keyboard or mouse. This is the final
+      step required to move away from dual booting with other operating systems
+      for legacy programs that require high performance graphics.
+    '';
+    homepage = https://looking-glass.hostfission.com/;
+    license = licenses.gpl2Plus;
```

Comment:
Do you have a source for this? The website only seems to talk about GPL 2:

https://looking-glass.hostfission.com/

Reply:
The [main.c file](https://github.com/gnif/LookingGlass/blob/06e38d897d2dfb76afc53070a1bf1833f6924649/client/main.c) of the client says "either version 2 of the License, or (at your option) any later version", so I assumed that meant GPL-2.0+.